python/tvm/micro/testing/utils.py

```diff
@@ -17,13 +17,16 @@
 
 """Defines the test methods used with microTVM."""
 
+import io
 from functools import lru_cache
 import json
 import logging
 from pathlib import Path
 import tarfile
 import time
 from typing import Union
+import numpy as np
+import pathlib
```

Comment:
Instead of re-importing this, just use `Path` directly which is already imported above.

Reply:
done